Podfile

```diff
@@ -1,12 +1,22 @@
 platform :ios, '8.0'
 source 'https://github.com/CocoaPods/Specs.git'
-link_with ["Signal", "SignalTests"]
+target 'Signal' do
+    pod 'SignalServiceKit',           :git => 'https://github.com/WhisperSystems/SignalServiceKit.git'
+    pod 'OpenSSL',                    '~> 1.0.205'
+    pod 'PastelogKit',                '~> 1.3'
+    pod 'FFCircularProgressView',     '~> 0.5'
+    pod 'SCWaveformView',             '~> 1.0'
+    pod 'DJWActionSheet'
 
-pod 'SignalServiceKit',           :git => 'https://github.com/WhisperSystems/SignalServiceKit.git'
-pod 'OpenSSL',                    '~> 1.0.205'
-pod 'PastelogKit',                '~> 1.3'
-pod 'FFCircularProgressView',     '~> 0.5'
-pod 'SCWaveformView',             '~> 1.0'
-pod 'DJWActionSheet'
+    pod 'JSQMessagesViewController',  :git => 'https://github.com/WhisperSystems/JSQMessagesViewController', :commit => 'e5582fef8a6b3e35f8070361ef37237222da712b'
+end
+target 'SignalTests' do
```

Comment:
I think it would make way more sense for `SignalTests` to be a target within `Signal` as shown in [CocoaPods spec](https://guides.cocoapods.org/syntax/podfile.html#target).